### PR TITLE
Replace sample.get with sample.getOrElse

### LINKED ISSUE #825: Replace sample.get with sample.getOrElse in tests
Some tests are using `sample.get` from `ScalaCheck` generators, which could potentially return a `None` (in most cases this is not the intended behavior). Replace these with `sample.getOrElse` and provide a default value instead of `None`. 

Example of usage [here](https://github.com/cfpb/hmda-platform/blob/master/api/src/test/scala/hmda/api/http/public/InstitutionSearchPathSpec.scala#L32)

### PR DESCRIPTION
**Pull request by nickgrippin**
Closes #825

### CODE REVIEW

**Comment by jmarin** [comment]
Travis is reporting a compilation failure

File: query/src/test/scala/hmda/query/repository/filing/FilingComponentSpec.scala [hunk 1]
```diff
@@ -99,10 +99,10 @@ class FilingComponentSpec extends AsyncWordSpec with MustMatchers with FilingCom
     "Stream rows for a specific respondent id" in {
       val respId = "resp2"
       val p = ""
-      val lar1 = larGen.sample.get.copy(respondentId = respId)
-      val lar2 = larGen.sample.get.copy(respondentId = respId)
-      val lar3 = larGen.sample.get.copy(respondentId = respId)
-      val lar4 = larGen.sample.get.copy(respondentId = "resp3")
+      val lar1 = larGen.sample.getOrElse(LoanApplicationRegister()).copy(respondentId = respId)
```
Comment (schbetsy) [inline query/src/test/scala/hmda/query/repository/filing/FilingComponentSpec.scala]: Code like this is repeated lots of times: `larGen.sample.getOrElse(LoanApplicationRegister())`, `InstitutionGenerators.institutionGen.sample.getOrElse(Institution.empty)`, etc.

Could it work to abstract that into a method? Like this: `LarGenerators.getOne` and `InstitutionGenerators.getOne`

**Comment by schbetsy** [comment]
Looking again, I think a more intuitive name for those methods would be `sampleLar` and `sampleInstitution`. 

Otherwise, this looks good to me. 👍

**Comment by jmarin** [comment]
LGTM

**Pull request merged**